docs(color): DLT-2592 theming guidelines

jcarter-dp opened the pull request:
Create the initial page, none of the contents is written properly. Just a written template.

# docs(color): DLT-2592 theming design tokens

<!--- Feel free to remove any unused sections -->

## Obligatory GIF (super important!)

<!--- go to giphy.com, pick a gif, click share, copy gif link, paste within the () brackets below. -->

![Obligatory GIF](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExY2VlNzVuazdicGh1cDRmaDM1eTEyamtlbXlyajBuODhvZW05NWU0NCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/rCeIPA4u4xqk5JlU0k/giphy.gif)

## :hammer_and_wrench: Type Of Change

<!--- Tick or place an `x` for the type of change. Should match the type in the commit message / PR title
in https://github.com/dialpad/dialtone/blob/staging/.github/COMMIT_CONVENTION.md -->

These types will increment the version number on release:

- [ ] Fix
- [ ] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [ ] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket

<!--- Enter the URL of the Jira ticket associated with this PR -->
[DLT-2592](https://dialpad.atlassian.net/browse/DLT-2592)

## :book: Description

<!--- Describe specifically what the changes are -->

## :bulb: Context

<!--- Describe the purpose of the changes -->
<!--- Why did we make these changes? -->
<!--- What problem(s) do they solve? -->

## :pencil: Checklist

<!--- Tick or place an `x` in all of the checkboxes that apply -->
<!--- Remove checkboxes that do not apply -->

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

For all Vue changes:

- [ ] I have added / updated unit tests.
- [ ] I have made my changes in Vue 2 and Vue 3. Note: you may sync your changes from Vue 2 to Vue 3 (or vice versa) using the `./scripts/dialtone-vue-sync.sh` script. Read docs here: [Dialtone Vue Sync Script](../packages/dialtone-vue3/.github/CONTRIBUTING.md#dialtone-vue-sync-script)
- [ ] I have validated components with a screen reader.
- [ ] I have validated components keyboard navigation.

For all CSS changes:

- [ ] I have used design tokens whenever possible.
- [ ] I have considered how this change will behave on different screen sizes.
- [ ] I have visually validated my change in light and dark mode.
- [ ] I have used gap or flexbox properties for layout instead of margin whenever possible.

If new component:

<!--- There are lots of things to remember when adding a new component to the system! This is so you don't forget any of them. -->

- I am exporting any new components or constants:
  - [ ] from the index.js in the component directory.
  - [ ] from the index.js in the root (either `packages/dialtone-vue2` or `packages/dialtone-vue3`).
- [ ] I have added the styles for the new component to the `packages/dialtone-css` package.
- [ ] I have created a page for the new component on the documentation site in `apps/dialtone-documentation`.
- [ ] I have added the new component to `common/components_list.js`
- [ ] I have created a component story in storybook
- [ ] I have created story / stories for any relevant component variants in storybook
- [ ] I have created a docs page for the component in storybook.
- [ ] I have checked that changing all props/slots via the UI in storybook works as expected.

## :crystal_ball: Next Steps

<!--- Describe any future changes that need to be made after merging the PR, especially any follow up tasks after release. -->

## :camera: Screenshots / GIFs

<!--- Add these if necessary. Since we have deploy previews for every PR it may not always be. -->
<!--- Link any screenshots / GIFs below -->

## :link: Sources

<!--- Add any links to external reference material -->


[DLT-2592]: https://dialpad.atlassian.net/browse/DLT-2592?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

braddialpad commented on gulpfile.js:
```diff
@@ -30,17 +30,18 @@ const breakpoints = [
   '.d-w100p',
-  /^\.d-wmx(-(auto|unset)|[0-9]*(ch|p))$/m, // Max widths
+  /\.d-wmx(-(auto|unset)|[0-9]*(ch|p))$/, // Max widths
+  '.d-bgc-red-200',
```
This shouldn't be here right?

juliodialpad replied:
Ah, yeah, I used that for testing. Will remove it as we don't need it.

braddialpad commented:
These ones in the gulpfile are just for the docsite right? it has nothing to do with our published package?

juliodialpad commented:
> These ones in the gulpfile are just for the docsite right? it has nothing to do with our published package?

Right, it's the local config to our docsite, they're not included in the package.

jcarter-dp committed:
Initial page structure

Create the initial page, none of the contents is written properly. Just a written template.

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,209 @@
+
+### High Contrast Theme
+
+Enhanced accessibility theme with increased contrast ratios for users with visual impairments or challenging viewing conditions.
```
Remove for now.

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,209 @@
+
+## Overview
+
+Dialtone's theming system provides a flexible foundation for creating consistent visual experiences across different contexts, user preferences, and product variations. Our theming approach leverages semantic color tokens to enable seamless switching between light, dark, and custom brand themes while maintaining accessibility and design consistency.
```
Let's be sure we define "theme" very specifically, including paired with a visual. That is, the only themeable segment is the "shell" – top and left (at least for now).

francisrupert replied:
Let's also distinguish between `theme` and `mode`, i.e.

`mode`
* light
* dark
* high-contrast (future)

`theme`
* dp (default)
* tmo
* sunflower
* etc
* etc

francisrupert replied:
To that end... change the second and third dropdowns here...

<img width="1518" height="344" alt="Image" src="https://github.com/user-attachments/assets/473a2a34-9f09-4916-9e04-2f695a5c5360" />

...to...

<img width="1518" height="344" alt="Image" src="https://github.com/user-attachments/assets/ab183401-850b-4050-8c57-e71f709e304e" />

And log a Jira Work Item for DT Engineer to reflect this more specifically, i.e...

```html
<html 
  ...
  data-dt-theme="light"
  data-dt-brand="dp"
  ...
>
```
...changes to 
```html
<html 
  ...
  data-dt-theme="dp|tmo|sunflower|etc"
  data-dt-mode="light|dark|high-contrast(future)"
  ...
>
```
This would be a breaking change.

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
````diff
@@ -0,0 +1,209 @@
+  --dt-theme-accent-default: var(--dt-color-gold-500);
+  --dt-theme-accent-hover: var(--dt-color-gold-600);
+}
+```
````
Let's punt on doing this to a follow-up doc. We should discuss this more carefully.

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
````diff
@@ -0,0 +1,209 @@
+    /* Apply light theme tokens */
+  }
+}
+```
````
This isn't quite right, and should be authored by a DT engineer. It might already be covered elsewhere. Ask them.

jcarter-dp committed:
Interactive theme preview prototype

Added interactive theme preview to test themes in dialtone

jcarter-dp committed:
Update prototype

jcarter-dp committed:
Updated doc

-List of theme colors (wip)
- svg updates with dialtone tokens
- dos and donts accessibility contrast

jcarter-dp committed:
Updated svgs

francisrupert commented:
@jcarter-dp I'm assuming you should merge `staging` since @juliodialpad merged his https://github.com/dialpad/dialtone/pull/867. I'm guessing what was accomplished in that PR no longer blocks progress on this one.

jcarter-dp committed:
shell tokens test in doc page

juliodialpad committed:
Merge branch 'staging' into docs(color)-DLT-2592-theming-design-tokens

juliodialpad committed:
remove wrong dialtone import

jcarter-dp committed:
updates to svgs to use shell tokens

-update 2 of the svg images to use shell tokens
- color table update
-new svg for top area
-update backgrounds in svg

jcarter-dp committed:
Merge branch 'staging' into docs(color)-DLT-2592-theming-design-tokens

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+---
+title: Themes
+description: Create diverse themes for projects using our simplified theming infrastructure.
```
```suggestion
Theme options that allow user customization of Dialpad products.
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+
+Dialtone's theming system is a flexible foundation for creating consistent visual experiences. It lets you adapt the user interface to different contexts, preferences, and product variations. Our approach uses semantic color tokens to easily switch between themes and modes.
+
+Currently, the only themeable part of the application is the shell, which consists of the top and left navigation.
```
```suggestion
Theming is limited to the application's navigation regions, consisting of the top and left.
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+
+### Theme
+
+A theme represents a specific brand or product identity. It applies a unique set of colors to the shell, and it works in conjunction with a mode. For example, Dialpad default theme uses purple accents and T-mobile uses pink accents for the shell.
```
```suggestion
A theme represents the core color scheme. It applies a unique set of colors to themeable regions, and it works in conjunction with a mode. For example, the default "Dialpad" theme accent may be purple, while a "Sunflower" theme's is a deep yellow.
```

Don't want to make too much mention of T-Mobile."

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #1C1C1C;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #252525;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #F9F9F9;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #7C5CF6;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #7C5CF6;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #E6E6E6;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #1F1F1F;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #D4D4D4;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #B8A6FC;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #B8A6FC;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #1C1C1C;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #252525;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #F9F9F9;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #E20074;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #E20074;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #E6E6E6;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #1F1F1F;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #D4D4D4;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #F8A6CB;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #F8A6CB;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #380010;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #6A652A;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #FFF9E5;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #93173A;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #93173A;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #1F1C00;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #FE1F76;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #FFF2F3;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #FA8D62;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #FA8D62;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #003165;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #98DCFF;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #E8E9E9;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #FF1356;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #FF1356;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #052133;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #5FC4F9;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #F5F9FD;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #FF716F;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #FF716F;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #FFE6FD;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #190826;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #511E76;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #9A006A;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #9A006A;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #190826;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #FFE6FD;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #FFF0FE;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #FFB1CF;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #FFB1CF;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #D3E0F4;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #10022C;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #228782;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #651E06;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #651E06;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #072C2A;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #8CBAEB;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #8CBAEB;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #F4B59F;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #F4B59F;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #EDF0EB;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #1A2214;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #344626;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #C0CAAE;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #C0CAAE;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #C0CAAE;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #1A2214;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #3F5F01;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #EDF0EB;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #EDF0EB;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #E4E0E8;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #0A0029;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #1C1C1C;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #DE0276;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #DE0276;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #0A0029;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #10022C;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #65318E;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #E2A4E5;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #E2A4E5;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #F0E4CB;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #15242E;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #1B2F3B;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #F4CD0B;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #F4CD0B;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #282317;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #D2E0FF;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #ECF9F8;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #F4CD0B;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #F4CD0B;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #E6EBD8;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #022443;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #0D6D4C;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #6633BB;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #6633BB;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #072C2A;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #C6E4FF;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #D3BCFF;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #E6E6E6;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #E6E6E6;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #ECE2CE;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #141F00;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #223300;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #E45C10;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #E45C10;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+            <div class="color-swatch" style="background-color: #3F3D3C;" title="Background"></div>
+            <div class="color-swatch" style="background-color: #DCD138;" title="Sidebar"></div>
+            <div class="color-swatch" style="background-color: #F48662;" title="Border"></div>
+            <div class="color-swatch" style="background-color: #FAEBA4;" title="Brand"></div>
```
```suggestion
            <div class="color-swatch" style="background-color: #FAEBA4;" title="Accent"></div>
```

francisrupert commented on apps/dialtone-documentation/docs/design/colors/themes/index.md:
```diff
@@ -0,0 +1,351 @@
+
+### Shell Modifier Tokens
+
+These tokens, such as shell-action, shell-status, shell-core, and shell-accent, inherit their values from the shell base tokens. They use modifiers to create variations and specific color states (e.g., hover, active, disabled) for interactive elements within the topbar and sidebar.
```
```suggestion
These tokens, such as `shell-action...`, `shell-status...` `shell-mention...`, inherit their values from the shell base tokens. For example, they use modifiers to create variations or color states (e.g., `hover`, `active`, `disabled`).
```

francisrupert reviewed (approved):
Just some content suggestions, and we're good to go.

jcarter-dp committed:
Update apps/dialtone-documentation/docs/design/colors/themes/index.md

Co-authored-by: Francis Rupert <francis.rupert@dialpad.com>

jcarter-dp committed:
Update apps/dialtone-documentation/docs/design/colors/themes/index.md

Co-authored-by: Francis Rupert <francis.rupert@dialpad.com>

jcarter-dp committed:
Update apps/dialtone-documentation/docs/design/colors/themes/index.md

Co-authored-by: Francis Rupert <francis.rupert@dialpad.com>

jcarter-dp committed:
Update apps/dialtone-documentation/docs/design/colors/themes/index.md

Co-authored-by: Francis Rupert <francis.rupert@dialpad.com>

jcarter-dp committed:
Update apps/dialtone-documentation/docs/design/colors/themes/index.md

Co-authored-by: Francis Rupert <francis.rupert@dialpad.com>

jcarter-dp committed:
Update apps/dialtone-documentation/docs/design/colors/themes/index.md

Co-authored-by: Francis Rupert <francis.rupert@dialpad.com>

jcarter-dp committed:
Update apps/dialtone-documentation/docs/design/colors/themes/index.md

Co-authored-by: Francis Rupert <francis.rupert@dialpad.com>

jcarter-dp committed:
Update apps/dialtone-documentation/docs/design/colors/themes/index.md

Co-authored-by: Francis Rupert <francis.rupert@dialpad.com>

jcarter-dp committed:
Update apps/dialtone-documentation/docs/design/colors/themes/index.md

Co-authored-by: Francis Rupert <francis.rupert@dialpad.com>

jcarter-dp committed:
Update apps/dialtone-documentation/docs/design/colors/themes/index.md

Co-authored-by: Francis Rupert <francis.rupert@dialpad.com>

jcarter-dp committed:
Fix YAML syntax error
